versioned_docs/version-8.7/self-managed/setup/guides/configure-elastcisearch-prefix-indices.md

```diff
@@ -57,8 +57,6 @@ operate:
     env:
       - name: CAMUNDA_OPERATE_ELASTICSEARCH_INDEXPREFIX
         value: custom-operate
```

Comment:
This should be removed as well

Reply:
@jessesimpson36 Aaaah, I got it now, I didn't see the section you mentioned in the GH UI.
Good catch, and thank you :raised_hands: